src/affordance/tasks/sequential_interpreter.py

```diff
@@ -57,7 +59,9 @@ def shorten_prefix(self, prefix: str, to_remove: int) -> str:
         # split along demonstrations
         individual_programs = prefix.split("\n----\n")
 
-        shifted_programs = individual_programs[: -(to_remove + 1)] + [individual_programs[-1]]
+        shifted_programs = individual_programs[: -(to_remove + 1)] + [
```

Comment:
looks like I forgot to mention that I've been using `black --line-length 100`, which allows these lines to be longer. I'll fix that